client/tests/ReviewRating.test.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
```

Comment:
Are we going to use this?

Reply:
We might use it as we will write more tests later on.